app/src/main/java/com/activities/DeleteUserActivity.java

```diff
@@ -0,0 +1,103 @@
+package com.activities;
+
+import android.os.Bundle;
+import android.util.TypedValue;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TableLayout;
+import android.widget.TableRow;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.R;
+import com.common.ToastWrapper;
+import com.libs.globaldata.GlobalData;
+import com.libs.globaldata.ModelObject;
+import com.libs.globaldata.userdatabase.UserDatabase;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeleteUserActivity extends AppCompatActivity {
+    private final int textSize = 18;
```

Comment:
This value is not normalizaed to sp, so it will look differently on different screens.
See: https://stackoverflow.com/questions/2025282/what-is-the-difference-between-px-dip-dp-and-sp and the explanation of "sp".